metadata.mk

```diff
@@ -14,8 +14,9 @@ K8S_VERSION=v1.29.7
 COREDNS_VERSION=1.5.2
 ETCD_VERSION=v3.5.6
 HELM_VERSION=v3.11.3
-KINDEST_NODE_VERSION=v1.29.7
-KIND_VERSION=v0.22.0
+KINDEST_NODE_VERSION=v1.29.2
+KINDEST_NODE_VERSION_CAPZ=v1.28.9
```

Comment:
> I'm thinking of disassociating the kind version used in capz from the general kind version in [metadata.mk](http://metadata.mk/),

Agree. I feel we should keep all capz configurations to `process/testing/winfv-felix/capz/` directory. It would be easier to maintain.